crates/cast/tests/e2e/main_tests.rs

```diff
@@ -119,7 +120,20 @@ async fn test_happy_case_mixed() {
 
 #[tokio::test]
 async fn test_missing_account() {
+    let temp_dir = TempDir::new().expect("Unable to create a temporary directory");
+    let manifest_path = temp_dir.path().join("./Scarb.toml");
+    let manifest_path = manifest_path.to_str().unwrap();
+
+    fs_extra::file::copy(
+        "tests/data/files/noconfig_Scarb.toml",
+        manifest_path,
+        &fs_extra::file::CopyOptions::new().overwrite(true),
+    )
+    .unwrap();
```

Comment:
maybe we could use `crate::helpers::fixtures::duplicate_directory_with_salt` instead? 
if not suitable let's add another helper function that does this, as the code is repeated in multiple places.

Reply:
Moved to a new utility function